src/model_bias_analysis.py

```diff
@@ -258,3 +254,34 @@ def per_term_negative_rates(df, terms, model_families, threshold, text_col,
             })
         records.append(term_record)
     return pd.DataFrame(records)
+
+
+### Plotting.
+
+def per_term_scatterplots(df, term_col, values_col, title='', y_lim=(0.8, 1.0),
```

Comment:
Wow! Given how nice the plots look, I thought this would be more complicated. Do you get the nice colors for free?

Reply:
Yep. Matplotlib defaults. I think they made the defaults much nicer with their 2.0 release.